src/controllers/profile.ts

```diff
@@ -1,25 +1,72 @@
+import dotenv from "dotenv";
 import { Request, Response } from "express";
 import { jwtDecode } from "jwt-decode";
 import Profile from "../models/profileModel";
 import { dateToZodiac } from "../services/dateToZodiac";
 import { setLocation } from "../services/location";
+import { ok } from "assert";
+import fs from "fs";
+
+dotenv.config();
+
+const cloudinary = require('cloudinary').v2;
 
 export const registerProfile = async (req: Request, res: Response) => {
+  cloudinary.config({
+    cloud_name: process.env.CLOUDINARY_NAME,
+    api_key: process.env.CLOUDINARY_API_KEY,
+    api_secret: process.env.CLOUDINARY_SECRET,
+  });
   const { name, dateOfBirth, location, reasons, gender } = req.body;
+  const preferences = JSON.parse(req.body.preferences);
   const token = req.headers.authorization?.split(" ")[1];
   const decodedToken: any = jwtDecode(token!);
   const userId = decodedToken.sub;
+  // Массив для хранения ссылок на загруженные файлы
```

Comment:
this comment is not really helpful, you can remove it

Reply:
+done